Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/urls.ts

```diff
@@ -1,26 +1,25 @@
 // TODO: https://github.com/SoftUni-Internal/exam-systems-issues/issues/133
-const baseUrl = window.URLS.UI_URL;
-const administrationBaseUrl = window.URLS.ADMINISTRATION_URL;
 
-const loginSubmitUrl = `${baseUrl}/Account/Login`;
-const logoutUrl = `${baseUrl}/Account/Logout`;
+const loginSubmitUrl = `${window.URLS.UI_URL}/Account/Login`;
+const logoutUrl = `${window.URLS.UI_URL}/Account/Logout`;
 
-const getProfileInfoUrl = `${baseUrl}/Users/GetProfileInfo`;
+const getProfileInfoUrl = `${window.URLS.UI_URL}/Users/GetProfileInfo`;
 
-const getIndexContestsUrl = `${baseUrl}/Contests/GetForHomeIndex`;
-const startContestParticipationUrl = `${baseUrl}/Compete/Index/%id%?official=%official%`;
+const getIndexContestsUrl = `${window.URLS.UI_URL}/Contests/GetForHomeIndex`;
+const startContestParticipationUrl = `${window.URLS.UI_URL}/Compete/Index/%id%?official=%official%`;
 
-const getSubmissionDetailsUrl = `${baseUrl}/Submissions/Details`;
-const getSubmissionsForProfileUrl = `${baseUrl}/Submissions/GetForProfile`;
-const submitUrl = `${baseUrl}/Compete/Submit`;
+const getSubmissionDetailsUrl = `${window.URLS.UI_URL}/Submissions/Details`;
+const getSubmissionsForProfileUrl = `${window.URLS.UI_URL}/Submissions/GetForProfile`;
+const submitUrl = `${window.URLS.UI_URL}/Compete/Submit`;
 
-const getParticipationsForProfileUrl = `${baseUrl}/Participations/GetForProfile`;
+const getParticipationsForProfileUrl = `${window.URLS.UI_URL}/Participations/GetForProfile`;
 
-const downloadProblemResourceUrl = `${baseUrl}/ProblemResources/GetResource/%id%`;
+const downloadProblemResourceUrl = `${window.URLS.UI_URL}/ProblemResources/GetResource/%id%`;
 
-const getSubmissionResultsByProblem = `${baseUrl}/Submissions/GetSubmissionResultsByProblem/%id%?isOfficial=%isOfficial%&take=%take%`;
+const getSubmissionResultsByProblem = `
+${window.URLS.UI_URL}/Submissions/GetSubmissionResultsByProblem/%id%?isOfficial=%isOfficial%&take=%take%`;
 
-const administrationContestsGridUrl = `${administrationBaseUrl}/Contests`;
+const administrationContestsGridUrl = `${window.URLS.ADMINISTRATION_URL}/Contests`;
```

Comment:
Why is this change necessary?

Reply:
Due to the ESLint rule. I reverted the changes and suppressed the rule.